content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -57,9 +57,6 @@ has finished.
 Here's an example of a Job with two containers, one of which is a sidecar:
 
 {{% code_sample language="yaml" file="application/job/job-sidecar.yaml" %}}
-By default, this feature is not available in Kubernetes. To avail this feature, you
-need to enable the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-named `SidecarContainers`.
```

Comment:
This change needs to be verified as the related issue states that beta APIs are on by default but as of https://github.com/kubernetes/enhancements/issues/3136 , beta APIs are off by default

Reply:
L20-21 should also be update to reflect what @matthyx had suggested in https://github.com/matthyx/website/commit/29dff23d39e4b855cd32ba6db98cc703645108af

It looks like this change originally in `init-containers.md` was lost when `sidecar-containers.md` was added. It is important as at first glance the current wording gives the impression that sidecars containers are still alpha and need a feature flag set.

```
(Before)
Starting with Kubernetes 1.28 in alpha, a feature gate named `SidecarContainers`...

(After)
Enabled by default with Kubernetes 1.29, a feature gate named `SidecarContainers`
````

Reply:
@skaegi it is a good catch, we're indeed in beta
@reylejano the feature is ON by default:
https://github.com/kubernetes/kubernetes/blob/v1.29.0/pkg/features/kube_features.go#L1195

Reply:
@skaegi, @matthyx,
Thanks for the inputs.
I have updated the PR with suggestions https://github.com/matthyx/website/commit/29dff23d39e4b855cd32ba6db98cc703645108af.